.travis.yml

```diff
@@ -20,17 +20,14 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.6.*
```

Comment:
I suggest removing 2.5 and 2.6 here, as they reached end of maintenance. This will avoid making the Travis build too big.